openedx/adg/lms/applications/views.py

```diff
@@ -274,6 +274,7 @@ class CoverLetterView(RedirectToLoginOrRelevantPageMixin, View):
     """
 
     template_name = 'adg/lms/applications/cover_letter.html'
+    login_url = '/register'
```

Comment:
The view is using a mixin in which the dispatch function calls get_login_url() and redirects to to the url returned if the user in unauthenticated. According to the requirements the an unauthenticated user is to be redirected to signup hence, I had to redefine this attribute (login_url)

Reply:
I'm talking about the variable name. Instead of `login_url` it should be `registration _url` because your are storing registration url in it. If the method `get_login_url()` is misleading, it should also be updated to `get_registration_url`.